src/App.js

```diff
@@ -1,23 +1,30 @@
 import logo from './logo.svg';
 import './App.css';
 
+const todoList = [
+  {
+    title: "Complete Task 1",
+    id: 1
+  },
+  {
+    title: "Complete Task 2",
+    id: 2
+  },
+  {
+    title: "Complete Task 3",
+    id: 3
+  }
+];
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function (item) {
+          return (<li key={item.id}>{item.title}</li>)
+        })}
+      </ul>
```

Comment:
Nice job here!

Maybe try an implicit return and use an arrow function next time. It makes the code look a bit cleaner. Something like this:
`todoList.map(item => (<li key={item.id}>{item.title}</li>))`

Let me know if this doesn't make sense. I can explain further

Reply:
Thanks! I'll be using arrow functions next time.